clients/pkg/promtail/targets/lokipush/pushtarget.go

```diff
@@ -92,7 +94,16 @@ func (t *PushTarget) run() error {
 	}
 
 	t.server = srv
-	t.server.HTTP.Handle("/loki/api/v1/push", http.HandlerFunc(t.handle))
+
+	switch t.config.Format {
+	// NDJSON and Plaintext are both newline delimited formats and will be handled the same.
+	case "ndjson", "plaintext":
```

Comment:
Now seeing that we're going for different path I'm wondering if we shouldn't just activate both and not have a format config WDYT ?

Reply:
We can do that for sure. Is the path ok?

Reply:
Yes let's activate both.

I would like `/promtail/api/v1` to be `/promtail/api/v1/raw`